bin/sl-run.js

```diff
@@ -18,6 +18,28 @@ var agentOptions = {
   logger: config.logger,
 };
 
+var USE_PROBES = true;
+
+if (config.enableTracing) {
+  var tracer = require('../lib/tracer');
+  var traceObject = require('../lib/trace-object');
+  traceObject.tracerOptions.USE_PROBES = USE_PROBES;
+  traceObject.tracerOptions.wrapRedis = true;
+  traceObject.tracerOptions.wrapMysql = true;
+  traceObject.tracerOptions.wrapMongodb = true;
+  traceObject.tracerOptions.wrapMemcached = true;
+  traceObject.tracerOptions.wrapPg = true;
+  tracer(traceObject.tracerOptions);
+}
+
+var coreTracer = tracer().tracer;
+agentOptions.strongTracer = coreTracer;
```

Comment:
I would inject the top-level tracer, not just the .tracer property, so that agent has full access to the tracer.